test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceTests.cs

```diff
@@ -1980,7 +1980,7 @@ private SchemaCompareEndpointInfo CreateTestEndpoint(SchemaCompareEndpointType t
                 case SchemaCompareEndpointType.Project:
                     result.ProjectFilePath = comparisonObjectPath;
                     result.TargetScripts = targetScripts;
-                    result.DataSchemaProvider = "150";
+                    result.DataSchemaProvider = "160";
```

Comment:
Not 161?

Reply:
this is the sql server version (160 = sql server 2022), not DacFx version. A few tests were failing because they had sql projects targeting sql server 2019 being compared to a sql server 2022 database, which has some new BuiltIns that DacFx now understands in this updated version of DacFx.